docs/content/1.guide/4.storage.md

```diff
@@ -6,24 +6,35 @@ icon: ri:database-2-line
 
 Nitro provides a built-in storage layer that can abstract filesystem or database or any other data source.
 
-`useStorage()` is an instance of [createStorage](https://unstorage.unjs.io/usage) using the [memory driver](https://unstorage.unjs.io/drivers/memory).
+`useStorage()` is an instance of [createStorage](https://unstorage.unjs.io/usage) using the [memory driver](https://unstorage.unjs.io/drivers/memory) by default. It's an auto-imported utility, so you can use it anywhere in your handler.
 
-**Example:** Simple (in memory) operations
+By default, Nitro will also mount the current directory using the filesystem driver, using a prefix and where each folder serve to extend the prefix.
 
-```js
+- Every file will be under the `root` base.
+- Every source file will be under the `src` base. (By default, `src` is the `root` base)
+- Every build (inside .nitro) will be under the `build` base.
```

Comment:
It is only available for development only and I consider it kinda more advanced usage. We might move it to the last section in the page